packages/server/lib/routes.integration.test.ts

```diff
@@ -10,6 +10,35 @@ describe('route', () => {
     afterAll(() => {
         api.server.close();
     });
+
+    describe('Content-type', () => {
+        it.each(['GET', 'POST'] as const)('should enforce content-type %s', async (val) => {
```

Comment:
we support GET with body? 🙃

Reply:
It was mostly to ensure it works because we are always putting the header no matter the method (in the UI and probably other customers too)